DOCSP-31924 mongodb-client-encryption changes for 6.0 (#739)
mongodb/docs-node: source/upgrade.txt

  ``tlsCertificateKeyFile`` connection options when you call the
  ``MongoClient.connect()`` method, not when you create the
  ``MongoClient`` instance.

> - The major version number of ``mongodb-client-encryption`` must now be the
>   same as the major version number of the {+driver-short+}. For example,

Reviewer: Suggestion:

I think this could be more succinct and could emphasize the condition under which they need to match.

```suggestion
- If you add ``mongodb-client-encryption`` as a dependency,
   the major version number must match that of the {+driver-short}.
   For example,
```

  {+driver-short+} v6.x.x requires ``mongodb-client-encryption`` v6.x.x.

> - Automatic Encryption methods are now in the {+driver-short+}. You must now
>   import these methods from the driver rather than from

Reviewer: Suggestion:

I think the first sentence establishes that there's a change, so I would omit the second "now". 

 think "rather than" implies something is preferred less which conflicts with "must".

```suggestion
- Automatic Encryption methods are now in the {+driver-short+}. You must
  import these methods from the driver instead of from
```
